### ISSUE TITLE
Why did not use Conv2DTranspose rather than Conv2D in the generator 

### ISSUE THREAD

**Issue by lestel** [jacobgil/keras-dcgan#24]
the related paper all say they use Conv2DTranspose

**Comment by NagabhushanSN95** [comment]
Yes. That is one of their key contributions. 
They say to 
1. use strided convolutions instead of max-pooling layer
2. use cond2dTranspose instead of upsampling
3. use batch normalization layers
4. use relu activations for intermediate layers

@jacobgil  You've implemented none of those. So, this is not DC GAN right?
